MAUI/Rotator/Dots-Customization.md

```diff
@@ -0,0 +1,291 @@
+                xmlns:local="clr-namespace:Rotator"
+                x:Class="Rotator.Rotator">
+        <ContentPage.BindingContext>
+            <local:RotatorViewModel/>
+        </ContentPage.BindingContext>
+        <ContentPage.Content>
+            <syncfusion:SfRotator x:Name="rotator" 
+                                ItemsSource="{Binding ImageCollection}" 
+                                SelectedIndex="2"
+                                BackgroundColor="#ececec"
+                                NavigationStripPosition="Bottom"
+                                DotsBorderColor="Aqua"
+                                WidthRequest="550"
+                                HeightRequest="550">
+                <syncfusion:SfRotator.ItemTemplate>
+                    <DataTemplate>
+                        <Image  Source="{Binding Image}"/>
+                    </DataTemplate>
+                </syncfusion:SfRotator.ItemTemplate>
+            </syncfusion:SfRotator>
+        </ContentPage.Content>
+    </ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+    using Syncfusion.Maui.Core.Rotator;
+    using Syncfusion.Maui.Rotator;
+
+    namespace Rotator
+    {
+        public partial class Rotator : ContentPage
+        {
+            public Rotator()
+            {
+                InitializeComponent ();
+                SfRotator rotator = new SfRotator();
+                var ImageCollection = new List<RotatorModel> {
+                new RotatorModel ("image1.png"),
+                new RotatorModel ("image2.png"),
+                new RotatorModel ("image3.png"),
+                new RotatorModel ("image4.png"),
+                new RotatorModel ("image5.png")
+                };
+                var itemTemplate = new DataTemplate(() =>
+                {
+                    var grid = new Grid();
+                    var nameLabel = new Image();
+                    nameLabel.SetBinding(Image.SourceProperty, "Image");
+                    grid.Children.Add(nameLabel);
+                    return grid;
+                });
+                rotator.ItemTemplate = itemTemplate;
+                rotator.DotsBorderColor = Colors.Aqua;
+                rotator.ItemsSource = ImageCollection;
+                rotator.WidthRequest=550;
+                rotator.HeightRequest=550;
+                this.Content = rotator;
+            }
+        }
+        public class RotatorModel
+        {
+            public RotatorModel(string imageString)
+            {
+                Image = imageString;
+            }
+            private String _image;
+            public String Image
+            {
+                get { return _image; }
+                set { _image = value; }
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfRotator with dots border color](images/DotsBorderColor.png)
+
+## Selected Dot Color
+
+The `SelectedDotColor` property is used to customize the color of selected dots in the `SfRotator`.
```

Comment:
The `SelectedDotColor` property is used to customize the color of selected dot in the `SfRotator`.

Reply:
changed dots to dot